src/backend/github/middleware/middleware.test.ts

```diff
@@ -1,11 +1,11 @@
 import { logger } from "probot/lib/logger";
 import { mocked } from "ts-jest/utils";
-import { Installation, Subscription } from "../../../src/models";
-import GitHubAPI from "../../../src/config/github-api";
-import middleware from "../../../src/github/middleware";
-import { mockModels } from "../../utils/models";
+import { Installation, Subscription } from "../../models";
+import GitHubAPI from "../../../config/github-api";
+import middleware from ".";
+import { mockModels } from "../../../common/mocks/models";
```

Comment:
This is why I don't like tests mixed into production code: the `mockModels` are included in the production bundle. I think the build will not include any files ending with `.test.ts` into the bundle, but since this mock file doesn't have the `.test.ts` in the name, it will be included. That increases the bundle size unnecessarily and makes it easier to accidentally use test code in production code.

Reply:
It's not a front-end app.  The end compilation (you can do it yourself with `npm run release`) is separate files that are included with require.  If a random file gets compiled and not included, it just doesn't matter.  the 'bundle size' will maybe be kilobytes bigger, but the file itself will never be ran anyways.

There are extra checks we can do as well as part of tsc that checks for dangling files (files not required by anyone) which can break the build if not following conventions.  The only issue that could occur is if someone were to accidentally require a test file in a production file (which we can do right now) and there are other checks we can add in place to prevent this from happening.

All in all, I don't think it's an issue and will makes development/testing easier.